test/performance/latency/LatencyTestPublisher.cpp

```diff
@@ -781,8 +781,8 @@ bool LatencyTestPublisher::test(
 
                 if (!loaned)
                 {
-                    logError(LatencyTest, "Problem on Publisher test data with loan");
```

Comment:
Unless this happens often (and therefore, can be seen as 'normal'), I would keep the Error level log, or at least, Warning level (although not terminating the execution), as it alerts about something smelly on the test.

The same goes for the rest of logs modified in this commit.

Reply:
With the larger payload pool and the several trials strategy, these never log. Let's promote them to logError 👍